src/app/auth/auth.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../core/services/auth.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'qpac-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.scss']
+})
+export class AuthComponent implements OnInit {
+
+  constructor(
+    private auth: AuthService,
+    private router: Router
+  ) { }
+
+  ngOnInit() {
+  }
+
+  public doFbLogin() {
+    this.auth.doFbLogin().then(res => {
+      console.log(res);
```

Comment:
Any logs into console should be added only for dev debug, e.g. redux state. In this case, it's not needed.